app/crud.py

```diff
@@ -0,0 +1,52 @@
+from sqlalchemy.orm import Session
+from models import City, Temperature
+from schemas import CityCreate, TemperatureCreate
+from datetime import datetime
+
+# CRUD для міста
+def create_city(db: Session, city: CityCreate):
+    db_city = City(name=city.name, additional_info=city.additional_info)
+    db.add(db_city)
+    db.commit()
+    db.refresh(db_city)
+    return db_city
+
+def get_city(db: Session, city_id: int):
+    return db.query(City).filter(City.id == city_id).first()
+
+def get_cities(db: Session, skip: int = 0, limit: int = 10):
+    return db.query(City).offset(skip).limit(limit).all()
+
+def update_city(db: Session, city_id: int, city: CityCreate):
+    db_city = db.query(City).filter(City.id == city_id).first()
+    if db_city:
+        db_city.name = city.name
+        db_city.additional_info = city.additional_info
+        db.commit()
+        db.refresh(db_city)
+    return db_city
+
+def delete_city(db: Session, city_id: int):
+    db_city = db.query(City).filter(City.id == city_id).first()
+    if db_city:
+        db.delete(db_city)
+        db.commit()
+    return db_city
+
+# CRUD для температури

```

Comment:
[CLARIFICATION] This comment is unnecessary since the function's purpose related to temperature is already clear.